pyroute2/dhcp/enums/dhcp.py

```diff
@@ -1,4 +1,4 @@
-from enum import IntEnum
+from enum import EnumType, IntEnum
```

Comment:
`EnumType` was added in Python 3.11, while we declare compatibility up to 3.9.

Reply:
Found a way to do without !